src/SelectMulti.vue

```diff
@@ -425,7 +429,7 @@
 				</div>
 			</slot>
 			</div>
-			<div class="combo-input-icon-block" :class="{ 'not-clickable': !$slots['input-icon'] }">
+			<component :is="iconIsClickable ? 'button' : 'div'" @click="iconIsClickable ? updateMenuState(iconIsClickable) : false" class="combo-input-icon-block" :class="{ 'not-clickable': !iconIsClickable }">
```

Comment:
what does this `not-clickable` css class do ? 
Is it just for tests ? If yes I would prefer to follow practise with `data-testid` attribute